include/nuttx/spinlock.h

```diff
@@ -286,6 +304,7 @@ void spin_unlock_wo_note(FAR volatile spinlock_t *lock);
 
 /* bool spin_islocked(FAR spinlock_t lock); */
 #define spin_islocked(l) (*(l) == SP_LOCKED)
+#define spin_is_locked(l) spin_islocked(l)
```

Comment:
@xiaoxiang781216 in this case isn't it better just replace spin_islocked() with spin_is_locket() ? It is confusing because now we have two functions with similar names that do same thing! :-)

Reply:
OK

Reply:
done.